lib/internal/Magento/Framework/Setup/Declaration/Schema/SchemaConfig.php

```diff
@@ -72,10 +72,18 @@ public function getDbConfig()
      */
     public function getDeclarationConfig()
     {
-        $schema = $this->schemaFactory->create();
-        $data = $this->readerComposite->read(FileResolverByModule::ALL_MODULES);
-        $this->declarativeSchemaBuilder->addTablesData($data['table']);
-        $schema = $this->declarativeSchemaBuilder->build($schema);
-        return $schema;
+        try{
+            $schema = $this->schemaFactory->create();
```

Comment:
You are wrapping the whole function. Try to be more specific to what Exception you want to catch and what method is likely to throw it and wrap just that line.

Can this try/catch be done deeper in the other methods?

Reply:
Okay let me be more specific with the exception